Menus/MainMenu/MainMenu.cs

```diff
@@ -135,6 +141,18 @@ void Start()
         {
             CreateFaceButtons();
             UpdateToolButtons();
+            RegisterSpatialMenuElements();
+        }
+
+        private void RegisterSpatialMenuElements()
+        {
+            spatialMenuData.Add(new SpatialMenu.SpatialMenuData("Workspaces", "Open a workspace", m_WorkspaceSpatialMenuElements));
+            spatialMenuData.Add(new SpatialMenu.SpatialMenuData("Tools", "Select a tool", m_ToolsSpatialMenuElements));

```

Comment:
This is fine for now, but it does seem off given how cleanly actions have their spatial data integrated